client/styles.css

```diff
@@ -13,6 +13,26 @@
 /*
 
 =====================
+PROFILE
+=====================
+
+*/
+.profile-height {
+  height: 600px;
+}
```

Comment:
As much as possible, let's try to stick with percentages and ems to make things more responsive.  But it's not a big deal for now.